libfreerdp/common/assistance.c

```diff
@@ -974,26 +964,26 @@ int freerdp_assistance_parse_file_buffer(rdpAssistanceFile* file, const char* bu
 			return -1;
 
 		length = q - p;
-
 		r = (char*) malloc(length + 1);
 
 		if (!r)
 			return -1;
 
 		CopyMemory(r, p, length);
 		r[length] = '\0';
+		errno = 0;
+		{
```

Comment:
Why are you setting errno here ?

Reply:
to ensure no other function has set ERRNO (tried to do that consistently before calling  ```strto*```)

Reply:
hmm, comment gone? Ah well, I tried to set ```errno = 0``` before ```strto*``` calls wherever possible